File: MiniDOMWebCtrls/DMD/CQSPD/StatiLavPraticaHandler.vb
Imports minidom
Imports minidom.Forms
Imports minidom.Databases
Imports minidom.WebSite

Imports minidom.Finanziaria
Imports minidom.Sistema
Imports minidom.Anagrafica
Imports minidom.Web

Namespace Forms

    Public Class StatiLavPraticaHandler
        Inherits CBaseModuleHandler

        Public Sub New()
            MyBase.New(ModuleSupportFlags.SCreate Or ModuleSupportFlags.SEdit Or ModuleSupportFlags.SDelete Or ModuleSupportFlags.SDuplicate Or ModuleSupportFlags.SAnnotations)

        End Sub


        Public Overrides Function CreateCursor() As DBObjectCursorBase
            Return New CStatiLavorazionePraticaCursor
        End Function




    End Class


End Namespace

File: MiniDOMWebCtrls/DMD/CQSPD/StatiLavPraticaHandler.cs

namespace minidom.Forms
{
    public class StatiLavPraticaHandler : CBaseModuleHandler
    {
        public StatiLavPraticaHandler() : base(ModuleSupportFlags.SCreate | ModuleSupportFlags.SEdit | ModuleSupportFlags.SDelete | ModuleSupportFlags.SDuplicate | ModuleSupportFlags.SAnnotations)
        {
        }

        public override Databases.DBObjectCursorBase CreateCursor()
        {
            return new Finanziaria.CStatiLavorazionePraticaCursor();
        }
    }
}